roles/openstack_helm_magnum/tasks/main.yml

```diff
@@ -37,6 +37,19 @@
         name: "{{ openstack_helm_magnum_helm_release_name }}"
         namespace: "{{ openstack_helm_magnum_helm_release_namespace }}"
 
+- name: Deploy cluster api
+  kubernetes.core.k8s:
+    kubeconfig: /etc/kubernetes/admin.conf
+    state: present
+    template:
+      - path: 'templates/capi-bootstrap.yml.j2'
+      - path: 'templates/capi-control-plane.yml.j2'
+      - path: 'templates/capi-core.yml.j2'
+      - path: 'templates/capi-provider.yml.j2'
+      - path: 'templates/capi-rbac.yml.j2'
+  run_once: true
+  changed_when: false
```

Comment:
There are some fields changed whenever apply so need to set this false manually.

- CRDs include conversion.webhook.clientConfig.caBundle which is injected by cert-manager after deployment. So it is always different.
- Rules of aggregated clusterRoles are rendered after clusterRole is deployed. So it is always different.

Reply:
I think there might be a Kubernetes setting to ignore fields on change?  Or was that Terraform?

Reply:
TF has but ansible doesn't